numpy/random/src/dsfmt/dSFMT-benchmark.c

```diff
@@ -0,0 +1,43 @@
+/*
+ *
```

Comment:
This file can be dropped.  It was an internal file I used for perf testing the wrapped code.

Reply:
Maybe should keep this.

Reply:
can this become a part of `benchmark/bench_random.py` somehow?

Reply:
It really is for a different purpose -- it is to check the performance of the wrapped implementation with a pure C implementation.